sigmf-spec.md

```diff
@@ -356,26 +357,45 @@ object:
 | `hw`            | false    | string  | A text description of the hardware used to make the Recording.|
 | `dataset`       | false    | string  | The full filename of the dataset file this Metadata file describes.|
 | `trailing_bytes`| false    | uint    | The number of bytes to ignore at the end of a Non-Conforming Dataset file.|
+| `metadata_only` | false    | bool    | Indicates the Metadata file is intentionally distributed without the dataset.|
 | `geolocation`   | false    | GeoJSON `point` object | The location of the Recording system.|
 | `extensions`    | false    | array   | A list of JSON Objects describing extensions used by this Recording.|
 | `collection`    | false    | string  | The base filename of a `collection` with which this Recording is associated.|
 
 ##### The `dataset` Field
+
 The `core:dataset` field in the Global object is used to specify the dataset file that
 this Metadata describes. If provided, this string MUST be the complete filename of the
 dataset file, including the extension. The dataset file must be in the local directory,
 and this string MUST NOT include any aspects of filepath other than the filename.
 
-If this field is omitted, the dataset file MUST be a SigMF Dataset file (NOT a
-Non-Conforming Dataset), and MUST have the same base filename as the Metadata file and
-use the `.sigmf-data` extension.
+If a Recording does not have this field, it MUST have a compliant SigMF dataset (NOT
```

Comment:
capital D

Reply:
so Dataset is not capitalized in this document, should it be changed to a proper noun everywhere it is used in this context? Recording, Archive, Collection are (and schema is not which i think is correct).